docs/source/semantic_segmentation.mdx

```diff
@@ -0,0 +1,161 @@
+# Image classification
+
+Semantic segmentation datasets are used to train a model to classify every pixel in an image. There are
+a wide variety of applications enabled by these datasets such as background removal from images, stylizing
+images, or scene understanding for autonomous driving. This guide will show you how to apply transformations
+to an image segmentation dataset.
+
+Before you start, make sure you have up-to-date versions of `albumentations` and `cv2` installed:
+
+```bash
+pip install -U albumentations opencv-python
+```
+
+This guide uses the [Scene Parsing](https://huggingface.co/datasets/scene_parse_150) dataset for segmenting
+and parsing an image into different image regions associated with semantic categories, such as sky, road, person, and bed.
+
+Load the dataset and take a look at an example:
+
+```py
+>>> from datasets import load_dataset
+
+>>> dataset = load_dataset("scene_parse_150")
+>>> index = 10
+>>> dataset["train"][index]
+{'image': <PIL.JpegImagePlugin.JpegImageFile image mode=RGB size=683x512 at 0x7FB37B0EC810>,
+ 'annotation': <PIL.PngImagePlugin.PngImageFile image mode=L size=683x512 at 0x7FB37B0EC9D0>,
+ 'scene_category': 927}
+```
+
+The dataset has three fields:
+
+* `image`: a PIL image object.
+* `annotation`: segmentation mask of the image.
+* `scene_category`: the label or scene category of the image (like “kitchen” or “office”).
+
+Next, check out an image with `dataset["train"][index]["image"]`:
```

Comment:
I think it'd be nice to put this code in a code block so users can easily copy it (same with the code below)